.github/workflows/nightly-audit.yml

```diff
@@ -0,0 +1,25 @@
+name: Nightly Security Cargo Audit
```

Comment:
isn't this already included in a separate PR on master?

Reply:
apologies - I didn't see this- yes it was included in master - let me resolve this